source/installation-guide/wazuh-dashboard/installation-assistant.rst

```diff
@@ -27,6 +27,8 @@ Wazuh dashboard installation
 
       # bash wazuh-install.sh --wazuh-dashboard dashboard
 
+   The default Wazuh web user interface port is 443, used by the Wazuh dashboard. You can specify this port by using the optional parameter ``-p|--port <port_number>``. Some recommended ports are 8443, 8444, 8080, 8888, and 9000.
```

Comment:
```suggestion
   The default Wazuh web user interface port is 443, used by the Wazuh dashboard. You can change this port using the optional parameter ``-p|--port <port_number>``. Some recommended ports are 8443, 8444, 8080, 8888, and 9000.
```

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/6322/commits/4425184ec74d403ea93b3ace210a69d469c2b00b.